README.md

```diff
@@ -224,7 +224,7 @@ There are three types of tests for Lantern and three corresponding commands:
 |  `make lint` | Runs the R and golang linters |
 |  `make lint_go` | Runs the golang lintr |
 |  `make lint_R` | Runs the R lintr |
-| `make json_export file=<export file name>` | Exports the history of the endpoint data to a JSON file specified by the 'file' parameter |
+| `make json_export file=<export file name> monthlyExport=<true/false>` | Exports the history of the endpoint data to a JSON file specified by the 'file' parameter. Setting monthlyExport equal to true creates the export file using only the last months history data, while setting it to false or leaving it blank will create the export file with all the history information from the last 30 days.  |
```

Comment:
Should it still be possible to create a json export file with all of the data if someone wants to? We do have the archive file so I'm thinking it's not super necessary, but I also don't think it would hurt to have the option available?